sale_automatic_workflow_validate_purchase_mto/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Sale Automatic Workflow Validate Purchase Mto',
+    'summary': """
+        When a sale order generates directly a purchase order,
+        validates it automatically""",
+    'version': '10.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'ACSONE SA/NV,Odoo Community Association (OCA)',
+    'website': 'https://acsone.eu',
```

Comment:
@sergio-teruel It's not mandatory in OCA guidelines

https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#modules